### DISCUSSION TITLE
ordering after ./gradlew generateMetadataByCountry

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#4581] [General]
@westnordost what happens where you run `./gradlew generateMetadataByCountry` ?

For me it consistently rearranges lists, so that `atmOperators` is after `orchardProduces`

But I see https://github.com/streetcomplete/StreetComplete/commit/f79e1e0ce5173ad7a1ec6e10766927d0e968494e - is it result of `./gradlew generateMetadataByCountry` ?

(if yes, then ideally something would be done so order is well defined and consistent across implementations)

**Comment by westnordost** [comment]
> For me it consistently rearranges lists, so that atmOperators is after orchardProduces

Not for me. Maybe it returns different results per platform? Feel free to add code to sort the results by alphabet before writing them to file
